File: Sakila_Net/VB_SakilaLib/Neg/NegFilmText.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegFilmText
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOFilmText.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( filmText as ClFilmText) as Boolean
                        Return DAO.DAOFilmText.SelectOne(filmText)     
                    End Function
                    Public  Shared Function crear( filmText as ClFilmText) as Integer
                        Return DAO.DAOFilmText.SqlInsert(filmText)     
                    End Function
                    Public  Shared Function eliminar( filmText as ClFilmText) as Integer
                        Return DAO.DAOFilmText.SqlDelete(filmText)     
                    End Function
                    Public  Shared Function actualizar( filmText as ClFilmText) as Integer
                        Return DAO.DAOFilmText.SqlUpdate(filmText)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  

File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegFilmText.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegFilmText
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOFilmText.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClFilmText filmText) 
        {
            return DAO.DAOFilmText.sqlSelect(filmText);
        }
        public  static int crear(ClFilmText filmText)
        {
            return DAO.DAOFilmText.sqlInsert(filmText);
        }
        public  static int eliminar(ClFilmText filmText)
        {
            return DAO.DAOFilmText.sqlDelete(filmText);
        }
        public  static int actualizar(ClFilmText filmText)
        {
            return DAO.DAOFilmText.sqlUpdate(filmText);
        }
    }
}
